cypress/integration/plugins/index-management-dashboards-plugin/aliases.js

```diff
@@ -26,11 +26,25 @@ describe("Aliases", () => {
   });
 
   beforeEach(() => {
+    // Intercept the specific POST request
+    cy.intercept("POST", "/api/ism/apiCaller", (req) => {
+      if (req.body.data && req.body.data.name === "**" && req.body.data.s === "alias:desc" && req.body.endpoint === "cat.aliases") {
+        req.alias = "apiCaller"; // Assign an alias directly if the condition is met
+      }
+    });
+
     // Visit ISM OSD
     cy.visit(`${BASE_PATH}/app/${IM_PLUGIN_NAME}#/aliases`);
 
-    // Common text to wait for to confirm page loaded, give up to 60 seconds for initial load
-    cy.contains("Rows per page", { timeout: 60000 });
+    // Wait for 120 seconds for OSD to start.
+    // eslint-disable-next-line cypress/no-unnecessary-waiting
+    cy.wait(120000);
+
+    // Wait for the API call to complete
+    cy.wait("@apiCaller", { timeout: 240000 });
```

Comment:
Can you add some log around this, so we get an idea how long it wait for the fist vist aliases call

Reply:
@bowenlan-amzn Done!